src/ray/core_worker/core_worker.h

```diff
@@ -51,6 +51,9 @@ class CoreWorker {
   /// \return Translated task language.
   ::Language ToTaskLanguage(WorkerLanguage language);
 
+  /// Initialize raylet client.
+  void InitializeRayletClient(int server_port);
```

Comment:
`server_port` -> `rpc_ server_port`, and document the argument?

Reply:
sure.